locations/models.py

```diff
@@ -77,6 +77,8 @@ def get_context(self, request):
                         if d <= self.max_dist_km:
                             locationsnearme.append(page)
         context['locationsnearme'] = locationsnearme
+        locationtagindexset=LocationTagIndexPage.objects.live().child_of(self)
+        context['tagspage']=locationtagindexset[0]
```

Comment:
this could probably be: `context['tagspage'] = LocationTagIndexPage.objects.live().child_of(self).first()`